Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -236,7 +236,7 @@ IPythonInterpreterFactory factory
                 Trace.TraceInformation($"Connection log: {_conn.LogFilename}");
             }
 
-            Task.Run(() => _conn.ProcessMessages().HandleAllExceptions(_services.Site, allowUI: false)).DoNotWait();
+            Task.Run(() => _conn.ProcessMessages().HandleAllExceptions(_services.Site, allowUI: false)).DoNotWait(waitInTests: false);
```

Comment:
I don't think we should trick the test. Have you figured out why `ProcessMessages` doesn't finish?

Reply:
Because the default analyzer doesn't shut down at the end of the test.

Reply:
This task would be better represented as a complete thread that we join on when disposing the analyzer (after it's terminated through another mechanism). It really shouldn't be going through `DoNotWait` at all.